cleaning_services/demo/sale_order.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="sale_order_1" model="sale.order">
+    <field name="partner_id" ref="res_partner_27"/>
+    <field name="sale_order_template_id" ref="sale_order_template_1"/>
+    <field name="prepayment_percent">1.0</field>
+    <!-- <field name="pricelist_id" ref="product_pricelist_1"/> -->
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="tag_ids" eval="[(6, 0, [ref('crm_tag_19'), ref('crm_tag_25'), ref('crm_tag_30')])]"/>
+    <field name="opportunity_id" ref="crm_lead_5"/>
+    <field name="starred_user_ids" eval="[(6, 0, [ref('base.user_admin')])]"/>
+    <field name="project_id" ref="industry_fsm.fsm_project"/>
```

Comment:
This shouldn't be defined on the SO, as it comes from the products on the SO lines
```suggestion
```